pkgs/build-support/portable-service/default.nix

```diff
@@ -0,0 +1,118 @@
+{ pkgs, lib, stdenv }:
+/*
+  Create a systemd portable service image
+  https://systemd.io/PORTABLE_SERVICES/
+
+  Example:
+  pkgs.portableService {
+      name = "demo";
+      units = [ demo-service demo-socket ];
+  }
+*/
+{
+  # The name and version of the portable service. The resulting image will be
+  # created in result/$name_$version.raw
+  name
+, version ? "dev"
```

Comment:
I am unsure about this and don't really want to have the default here while I can understand why you added it.

Reply:
ok. removed that. `version` is mandatory now.